services/brig/test/integration/API/Provider.hs

```diff
@@ -8,7 +8,7 @@ module API.Provider (tests, Config) where
 
 import Bilge hiding (accept, timeout, head)
 import Bilge.Assert
-import Brig.Types
+import Brig.Types hiding (NewPasswordReset (..), CompletePasswordReset(..), EmailUpdate (..), PasswordReset (..), PasswordChange (..))
```

Comment:
Oh dear, that's a lot of hiding. Did you try qualifying one of the imports instead?

Reply:
I thought the same... but importing qualified results in ~100 errors (either with `Brig.Types` or `Brig.Types.Provider`). Also importing only the needed types results in an even "uglier" import :/